docs/medical-api/more-info/webhooks.mdx

```diff
@@ -96,6 +96,16 @@ Example payload:
       ...
     }
   ]
+  "cxDocumentRequestMetadata": {
```

Comment:
I added both in the docs to this sample wh payload but it isn't actually the case that you could get both payloads back in one wh.